samples/ingestion/ingestion-client/Connector/ServiceBusUtilities.cs

```diff
@@ -8,26 +8,26 @@ namespace Connector
     using System;
     using System.Text;
     using System.Threading.Tasks;
-    using Microsoft.Azure.ServiceBus;
+    using Azure.Messaging.ServiceBus;
     using Microsoft.Extensions.Logging;
 
     public static class ServiceBusUtilities
     {
-        public static async Task SendServiceBusMessageAsync(QueueClient queueClient, string messageContent, ILogger log, TimeSpan delay)
+        public static async Task SendServiceBusMessageAsync(ServiceBusSender queueSender, string messageContent, ILogger log, TimeSpan delay)
         {
-            var message = new Message(Encoding.UTF8.GetBytes(messageContent));
-            await SendServiceBusMessageAsync(queueClient, message, log, delay).ConfigureAwait(false);
+            var message = new Azure.Messaging.ServiceBus.ServiceBusMessage(messageContent);
```

Comment:
```suggestion
            var message = new ServiceBusMessage(messageContent);
```

Reply:
@JoshLove-msft, There are ServiceBusMessage class in both Connector and Azure.Messaging.ServiceBus. So I need to distinguish them.